apollo/typedefs.ts

```diff
@@ -103,7 +102,7 @@ const typeDefs = gql`
   }
 
   type Organization {
-    id: ID!
+    id: String!
```

Comment:
why did you decide not to use the ID type anymore?

Reply:
I did not. Don't know why that ended up there. I changed it back to ID

Reply:
Hm. I don't know how to answer your first comment about signup. Sorry. I always forget that there is another page that might be affected by my changes. InputField now uses propName (a string or an array of strings) for what id and nesting used to do. It's now changed on signup page as well.